styles/globals.js

```diff
@@ -0,0 +1,49 @@
+export const hoverClass =
+  'flex flex hover:text-neutral-700 cursor-pointer duration-200';
+export const headerLinks = [
```

Comment:
can we move this from styles folder, it doesn't make sense to keep constants not related to styles here.
can be `util/constants.js` for example or can use similar to olas website approach `data/resources.json`